samples/sample_plugins/vpp_plugin/include/mfx_multi_vpp.h

```diff
@@ -32,32 +32,39 @@ class MFXVideoMultiVPP : public MFXVideoVPP
     virtual ~MFXVideoMultiVPP(void) { Close(); }
 
     // topology methods
-    virtual mfxStatus QueryIOSurf(mfxVideoParam *par, mfxFrameAllocRequest request[2], mfxVideoParam *par1 = NULL, mfxVideoParam *par2 = NULL)
-    { par1; par2; return MFXVideoVPP_QueryIOSurf(m_session, par, request); }
+    virtual mfxStatus QueryIOSurf(mfxVideoParam *par, mfxFrameAllocRequest *request) override { return QueryIOSurfMulti(par, request); }
+    virtual mfxStatus QueryIOSurfMulti(mfxVideoParam *par, mfxFrameAllocRequest request[2], mfxVideoParam* /*par1*/ = NULL, mfxVideoParam* /*par2*/ = NULL)
+    { return MFXVideoVPP_QueryIOSurf(m_session, par, request); }
 
-    virtual mfxStatus Init(mfxVideoParam *par, mfxVideoParam *par1 = NULL, mfxVideoParam *par2 = NULL)
-    { par1; par2; return MFXVideoVPP_Init(m_session, par); }
+    virtual mfxStatus Init(mfxVideoParam *par) override { return InitMulti(par); }
+    virtual mfxStatus InitMulti(mfxVideoParam *par, mfxVideoParam* /*par1*/ = NULL, mfxVideoParam* /*par2*/ = NULL)
+    { return MFXVideoVPP_Init(m_session, par); }
 
-    virtual mfxStatus Reset(mfxVideoParam *par, mfxVideoParam *par1 = NULL, mfxVideoParam *par2 = NULL)
-    { par1; par2; return MFXVideoVPP_Reset(m_session, par); }
+    virtual mfxStatus Reset(mfxVideoParam *par) override { return ResetMulti(par); }
+    virtual mfxStatus ResetMulti(mfxVideoParam *par, mfxVideoParam* /*par1*/ = NULL, mfxVideoParam* /*par2*/ = NULL)
```

Comment:
ok, this one looks to be completely same as ```Reset``` simply because you don't use additional parameters. Are there any components which really needs those ```Multi``` variants? Maybe we can remove additional parameters in functions calls and switch to regular ```Reset```/```Init```/```Query``` since we don't really need ```Multi``` variants?